docs/pages/includes/install-linux.mdx

```diff
@@ -1,22 +1,24 @@
-<ScopedBlock scope={["enterprise"]}>
-
-Visit your [Teleport account](https://teleport.sh) and select the URL for your package of choice.
-
-</ScopedBlock>
+Use the appropriate commands for your environment to install your package.
 
-<ScopedBlock scope={["cloud"]}>
+<Admonition type="note">
```

Comment:
My first impression when seeing this partial on the dev server is that it's quite a lot of visual information to take in. There are multiple layers of boxes, a code snippet, tabs, a dropdown menu, and a bulleted list. Is there a way to simplify the experience of viewing this partial?

<img width="924" alt="image" src="https://user-images.githubusercontent.com/25250668/236951734-12fbe1c6-3fa0-4e9a-ac94-396d4e0ba60b.png">

Reply:
I removed the bulleted list and put it on the `installation.mdx` page instead. Installation instructions now contain a link to this page instead.